.travis/cargo-test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
I like it how you used `bash -e` in your other scripts. Could we rephrase this one as well?